### PR TITLE
docs(jotai-valtio): add  API section to docs

### PR DESCRIPTION
**Pull request by dmaskasky**
## Related Issues or Discussions
Issue https://github.com/jotaijs/jotai-valtio/issues/4

## Summary
Introduce a new section in the  documentation
to describe the functionality, use-cases, and examples for the new  API in the jotai-valtio integration library.

## Check List

- [x] `yarn run prettier` for formatting code and docs

### CODE REVIEW

File: docs/integrations/valtio.mdx [hunk 1]
```diff
@@ -70,3 +70,51 @@ atomWithProxy(proxyObject, { sync: true })
...
+mutableAtom(value, options?)
+```
+
+**value** (required): the value to proxy.
+
+**options.proxyFn** (optional): allows customization with `proxyFn` for custom proxy functions. Can be `proxy` (default) or a custom function.
+
+### Examples
+
+<CodeSandbox id="f84sk5" />
```
Comment (dmaskasky) [inline docs/integrations/valtio.mdx:101]: What is needed to get this to work?

Full URL: https://codesandbox.io/s/morning-darkness-f84sk5

Comment (dai-shi) [reply docs/integrations/valtio.mdx:101]: What do you mean?

It seems working: https://jotai-git-fork-dmaskasky-docs-valtio-mutableatom-pmndrs.vercel.app/docs/integrations/valtio

Comment (dmaskasky) [reply docs/integrations/valtio.mdx:101]: Oh nice.

**Review by dai-shi — APPROVED** [review]
Thanks!

**Pull request merged**